src/modules/testimonials/dto/create-testimonial.dto.ts

```diff
@@ -0,0 +1,35 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsString } from 'class-validator';
+
+export class CreateTestimonialDto {
+  @IsString()
+  @IsNotEmpty()
+  @ApiProperty({
+    type: String,
+    description: 'Name of the user',
+    example: 'John Doe',
+  })
+  name: string;
+
+  @IsString()
+  @IsNotEmpty()
+  @ApiProperty({
+    type: String,
+    description: 'Content of the testimonial',
+    example: 'I am very happy with the service provided by the company',
+  })
+  content: string;
+}
+
+export class CreateTestimonialResponseDto {
+  'status': string;
+  'message': string;
+  'data': TestimonialData;
+}
+
+interface TestimonialData {
```

Comment:
export this to a new file as well, probably in an interface folder, and rename to TestimonialInterface or TestimonialDataInterface

Reply:
Moved